app/institutions/dashboard/-components/object-list/contributors-field/component.ts

```diff
@@ -12,21 +13,30 @@ export default class InstitutionalObjectListContributorsField extends Component<
     // Return two contributors affiliated with the institution given with highest permission levels
     get topInstitutionAffiliatedContributors() {
         const { searchResult, institution } = this.args;
-        const contributors: any[] = searchResult.resourceMetadata.creator;
+        const contributors: any[] = searchResult.resourceMetadata.qualifiedAttribution;
         const institutionIris = institution.iris;
         const affiliatedContributors = contributors
             .filter((contributor: any) => hasInstitutionAffiliation(contributor, institutionIris));
-        // TODO: get the two users with the highest permission level and add permission to the return object
-        return affiliatedContributors.slice(0, 2).map((contributor: any) => ({
-            name: contributor.name[0]['@value'],
-            url: contributor['@id'],
+        const adminContributors = affiliatedContributors
+            .filter(contributor => contributor.hadRole[0]['name'][0]['@value'] === AttributionRoles.Admin);
```

Comment:
A question on the expected number of members in the `hadRole` property. If I am an admin contributor for a project, should I expect an entry in the `hadRole` for admin, write, and read-only, similar to `current_user_permissions`? Or would having the `osf:admin-contributor` imply that I have all of the lesser permissions associated

Reply:
hmm, i was thinking osf permission would map to only one role (similar to [how it shows on `contributors` in the api](https://api.osf.io/v2/nodes/ezcuj/contributors/?fields%5Bcontributors%5D=permission)), but i'd be open to changing it to include (up to) all three, might make more sense -- in that case would want to rename `osf:readonly-contributor` to `osf:read-contributor`, because it's no longer "only"

either way has a bit of confusion because permissions and roles aren't quite the same thing, but oh well

Reply:
Ah, I forgot that node-contributors only showed the highest level permission... As you mentioned, I conflated "role" and "permissions", and upon reflection, I think it's reasonable to think that a property called "hadRole" (singular) would show just one value. My preference would be to keep it as it is, with just one role, unless something else comes up in the future.

I was mostly curious if this `.filter`'s callback function could just do the simple `[0]['@id'] === osf:admin-contributor` or if it would need to do a `.filter(role => role['@id'] === osf:admin-contributor` (I will be working on refactoring this to use the utilities you suggested in this PR tomorrow, but wanted to update this first in case I run into issues with that refactor 🤞)

Reply:
heh, you'd kinda convinced me we _should_ do multiple values, to make it more explicit, but i'm good with keeping just one `osf:` role for the moment -- in terms of brittleness over time, tho, was thinking osf might have more `hadRole` values if it ever supports an actual [contributor role taxonomy](https://jats4r.niso.org/credit-taxonomy/#table-1-credit-terms-and-urls)) (which allows and encourages multiple roles per person)

and generally, in rdf-based metadata, i'd like to encourage assuming there may be multiple values for any property, except in exceptional cases (hence all the obnoxious arrays...) -- plural predicate names are less of a thing, because the underlying data model is a set of triple-statements that each has only one object and makes sense with a singular predicate name, e.g.:
```
:blarg prov:hadRole credit:conceptualization
:blarg prov:hadRole credit:software
:blarg prov:hadRole credit:validation
```
but the subject and predicate are deduplicated in turtle:
```
<:blarg> prov:hadRole <:role1>, <:role2>, <:role3>, <:role4> .
```
and in jsonld:
```
{
 "@id": ":blarg",
 "prov:hadRole": ["<:role1>", "<:role2>", "<:role3>", "<:role4>"]
}
```
(...all that said, could easily choose a plural name in the osfmap-json shorthand to make clear it may have multiple values, but we haven't been doing that so far...)